web/templates/beneficiaries/view.html

```diff
@@ -100,5 +101,42 @@
             </div>
         </div>
     </form>
+    <div class="card" style="width: 40rem">
+        <div class="card-body">
+            <div class="d-flex justify-content-middle">
+                <h1>Cases</h1>
+            </div>
+            <table
+                id="data"
+                class="table table-striped table-sm mx-auto text-center"
+            >
+                <thead>
+                    <tr>
+                        <th>ID</th>
+                        <th>title</th>
+                        <th>Purpose</th>
```

Comment:
Don't want purpose / contact details .. it will increase size of row.   just link/title/state is enough